impeller/renderer/backend/vulkan/command_buffer_cache.h

```diff
@@ -0,0 +1,93 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#pragma once
+
+#include <optional>
+
+#include "flutter/impeller/renderer/backend/vulkan/vk.h"
+
+namespace impeller {
+
+template <typename CommandBuffer>
+class CommandBufferCache {
```

Comment:
I would call this PassBindingsCache like the Metal backend. This class doesn't seem to cache command buffers at all.

Reply:
Done.

> This class doesn't seem to cache command buffers at all.

"Pass bindings" isn't really accurate either since it is caching anything that is added to the command buffer, not just bindings.  I agree, lets keep it consistent though.